charts/core-dump-handler/templates/openshift-scc.yaml

```diff
@@ -0,0 +1,43 @@
+{{- if .Values.serviceAccount.createScc }}
+
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  annotations:
+    kubernetes.io/description: "allows pod to run as root, privileged and run sysctl"
+    "helm.sh/hook": pre-install
+  name: {{ .Values.serviceAccount.sccName }}
+allowHostDirVolumePlugin: true
+allowHostIPC: true
+allowHostNetwork: true
+allowHostPID: true
+allowHostPorts: true
+allowPrivilegeEscalation: true
+allowPrivilegedContainer: true
+allowedCapabilities:
+- '*'
+allowedUnsafeSysctls:
+- '*'
```

Comment:
can we lock this down even further? I admit that was a very lazy attempt at copying the existing privilege scc

Reply:
The sysctls are:
kernel.core_pattern
kernel.core_pipe_limit
fs.suid_dumpable
We need to identify if any capabilities are also required to modify those tuneables

Reply:
yep that works, thanks for the suggestion.